src/bundle/EventListener/RequestAttributesListener.php

```diff
@@ -73,6 +92,26 @@ function (Repository $repository) use ($locationId) {
         return $location;
     }
 
+    /**
+     * @param mixed $contentId
```

Comment:
As far as I know `$contentId` will always be `int`.

Reply:
In `loadContent` method it is typed as mixed, but apparently it is integer in this case